solace-spring-cloud-starters/solace-spring-cloud-stream-starter/README.adoc

```diff
@@ -264,6 +264,19 @@ The maximum time to wait for all unacknowledged messages to be acknowledged befo
 +
 Default: `10000`
 
+batchMaxSize::
+The maximum number of messages per batch. +
+Only applicable when `batchMode` is `true`.
++
+Default: `255`
+
+batchTimeout::
+The maximum wait time in milliseconds to receive a batch of messages. If this timeout is reached, then the messages that have already been received will be used to create the batch. A value of `0` means wait forever. +
+Only applicable when `batchMode` is `true`. +
+This config option takes precedence over `polledConsumerWaitTimeInMillis` when batching is enabled.
```

Comment:
"takes precedence" suggests that if `batchTimeout` is not specified, then `polledConsumerWaitTimeInMillis` takes effect. `polledConsumerWaitTimeInMillis` does not apply when `batchMode=true`.
If the idea is to compare the two config options, maybe something like: "This config option is similar to `polledConsumerWaitTimeInMillis` when batching is disabled."